.github/workflows/snarkjs.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Check if directory path argument is provided
+if [ "$#" -lt 2 ]; then
+    echo "Usage: $0 <directory_path> <curve>"
+    exit 1
+fi
+
+DIR_PATH=$1
+CURVE=$2
+NEW_PTAU=false
+
+# Check for optional setup flag
+for arg in "$@"; do
+    if [[ $arg == "--new" ]]; then
+        NEW_PTAU=true
+    fi
+done
+
+# Create a basic test circuit.circom if it doesn't exist
+if [ ! -f "$DIR_PATH/circuit.circom" ]; then
+    echo "Generating a basic circuit.circom file for testing..."
+    cat > "$DIR_PATH/circuit.circom" <<EOL
+template Multiplier() {
+    signal input a;
+    signal input b;
+    signal output c;
+
+    c <== a * b;
+}
+
+component main = Multiplier();
+EOL
+fi
+
+# Create a basic input.json for testing
+if [ ! -f "$DIR_PATH/input.json" ]; then
+    echo "Generating a basic input.json file for testing..."
+    cat > "$DIR_PATH/input.json" <<EOL
+{
+    "a": 3,
+    "b": 11
+}
+EOL
```

Comment:
I am not sure what is this for. 
I think we need to test against the circuits generated by noname instead of circom.